src/crypto/yespower-bitzeny.c

```diff
@@ -0,0 +1,50 @@
+/*-
+ * Copyright 2009 Colin Percival
+ * Copyright 2013-2018 Alexander Peslyak
+ * Copyright 2018 bitzenyPlus Core Developers
```

Comment:
https://github.com/bitzenyPlus/BitZenyPlus/issues/1#issuecomment-409073031

Reply:
Looks like this file doesn't contain anything written by Colin or by me, so please drop our copyright statements from it.